src/app/components/developer/developer-view.jsx

```diff
@@ -93,8 +93,9 @@ function ChplDeveloperView(props) {
                   { status.reason
                     && (
                       <>
-                        <br />
+                        <Typography gutterBottom>
```

Comment:
There's no reason to have the `<>` above, if all of the data inside this element is in a single Typography element. I think this should be something like

```
&& (
<Typography gutterBottom>
{ status.reason }
</Typography>
)}